.github/workflows/hip.yml

```diff
@@ -52,6 +52,9 @@ jobs:
         export CXX=$(which clang++)
         export CC=$(which clang)
 
+        export LIBRARY_PATH=/opt/rocm/lib:${LIBRARY_PATH:-}
+        export LD_LIBRARY_PATH=/opt/rocm/lib:${LD_LIBRARY_PATH:-}
+        export LDFLAGS="-lhsa-runtime64 ${LDFLAGS}"
```

Comment:
@ax3l 

I already added the export of `LDFLAGS` to the Tuolumne profile file. Do you think we need to add the other two as well (`LIBRARY_PATH`, `LD_LIBRARY_PATH`)?

Asking because I still don't understand why exporting `LIBRARY_PATH` and `LD_LIBRARY_PATH` turned out to be needed now and not before - so not sure if it was just a CI environment issue here.

Reply:
This works as a quick-fix, but I think we need a CMake `find_package` call to be robust and portable.